README.md

```diff
@@ -82,21 +82,34 @@ Diplomat::Kv.get('foo/', recurse: true)
 # => [{:key=>"foo/a", :value=>"lorem"}, {:key=>"foo/b", :value=>"ipsum"}, {:key=>"foo/c", :value=>"dolor"}]
 ```
 
-### Services
+### Nodes
 
 #### Getting
 
-Looking up a service is easy as pie:
+Look up a node:
 
 ```ruby
 foo_service = Diplomat::Service.get('foo')
 # => #<OpenStruct Node="hotel", Address="1.2.3.4", ServiceID="hotel_foo", ServiceName="foo", ServiceTags=["foo"], ServicePort=5432>
 ```
-Or if you have multiple nodes per service:
+
+Get all nodes:
+
```

Comment:
I think you left off something here. There's no documentation for getting all nodes.